R/stat-quantile.r

```diff
@@ -73,7 +74,13 @@ StatQuantile <- ggproto("StatQuantile", Stat,
     }
     grid <- new_data_frame(list(x = xseq))
 
-    method <- match.fun(method)
+    # if method was specified as a character string, replace with
+    # the corresponding function
+    if (is.character(method)) {
```

Comment:
You need to protect this block with `is.character()`, and can you please use https://style.tidyverse.org/syntax.html#indenting style?

Reply:
Do you mean "you *don't* need ..."? I added `is.character()` in an earlier version that needed it but now it seems superfluous.

Reply:
Yup sorry.